emitter/redshift/emitter.go

```diff
@@ -16,11 +16,13 @@ import (
 type RedshiftEmitter struct {
 	AwsAccessKey       string
 	AwsSecretAccessKey string
+	IAMRole			   string
```

Comment:
Note: this naming may cause an upstream pull request to stumble. Given the conventions in this repo, I'd name it `AwsIamRole`. Blending in is a great way to ensure a low-friction merge.